cmd/server/main.go

```diff
@@ -47,6 +47,10 @@ import (
 	"go.temporal.io/server/temporal"
 )
 
+var (
+	cgoEnabled = false
```

Comment:
where is this variable used?

Reply:
Just a few lines bellow.

Reply:
I'm not sure I follow. Line 146 uses build.InfoData.CgoEnabled which is either defined in cgo.go or nocgo.go. I don't see where this valuable is used.

Reply:
Sorry, @yiminc. It is leftover from 1st implementation. Thanks for catching this!